README.md

```diff
@@ -1,4 +1,4 @@
-# Automated expense tracking from Israeli banks and credit cards
+# Automated Expense Tracking from Israeli Banks and Credit Cards
```

Comment:
Nope. The original capitalization is correct

Reply:
Not arguing. I thought in the title it can be different, but I don't trust myself.

Reply:
Not arguing. I thought in the title it can be different, but I don't trust myself.